dev/bots/run_command.dart

```diff
@@ -133,9 +142,12 @@ Future<void> runCommand(String executable, List<String> arguments, {
     if (failureMessage != null) {
       print(failureMessage);
     }
-    if (!printOutput) {
-      stdout.writeln(utf8.decode((await savedStdout).expand<int>((List<int> ints) => ints).toList()));
-      stderr.writeln(utf8.decode((await savedStderr).expand<int>((List<int> ints) => ints).toList()));
+
+    // Print the output when we get unexpected results (unless output was
+    // printed already).
+    if (outputMode != OutputMode.print) {
```

Comment:
This should be a switch statement in case we end up adding more cases to the enum